server/.prettierrc

```
@@ -0,0 +1,7 @@
+{
+  "semi": true,
+  "trailingComma": "none",
+  "singleQuote": true,
+  "printWidth": 80,
+  "bracketSpacing": false
+}
```

```suggestion
{
  "printWidth": 80,
  "tabWidth": 2,
  "useTabs": false,
  "semi": true,
  "singleQuote": false,
  "quoteProps": "as-needed",
  "trailingComma": "none",
  "bracketSpacing": true,
  "bracketSameLine": false,
  "arrowParens": "always",
  "proseWrap": "preserve",
  "endOfLine": "lf",
  "embeddedLanguageFormatting": "auto",
  "singleAttributePerLine": false
}
```

I went through the default options on https://prettier.io/docs/en/options.html and built this config to the lastest defaults

Apart from "trailingComma" which is default "all" but in this case we set to "none"

We have all been using the Prettier VSCode Extension from the start of the course and that defaults to double quotes, so we are all used to coding with double quotes

**Could you run the `npm run prettier` command again on the PR so it removes the git diff for the single/double quotes on all the `.ts` files**